### ISSUE TITLE
Normalize HTML attribute order

### ISSUE THREAD

**Issue by Crissov** [babelmark/babelmark.github.io#3]
It would be helpful if Babelmark canonically reordered generated HTML attributes so that implementations that only differed in this regard were shown collectively.

Example
----------

~~~markdown
![foo](bar "baz")
~~~

Most implementations generate

~~~html
<img src="bar" alt="foo" title="baz">
~~~

but Rdiscount and Pandoc put `title` before `alt`:

~~~html
<img src="bar" title="baz" alt="foo">
~~~

and Python-Markdown puts `alt` first.

~~~html
<img alt="foo" src="bar" title="baz">
~~~

[live test](https://babelmark.github.io/?text=!%5Bfoo%5D(bar+%22baz%22))

**Comment by xoofx** [comment]
Indeed. Don't know when I can work on this but I will keep it around. thanks

**Comment by Crissov** [comment]
Add to this: ignore custom `data-*` attributes (e. g. Github).

**Comment by Crissov** [comment]
I've looked into the JS to find out where you normalize the HTML until I finally figured out that this is done on the [babelmark-proxy](/babelmark/babelmark-proxy) server-side already, i.e. the JSON returned by the Azure server, e.g. for <https://babelmark.azurewebsites.net/api/get?text=foo>, already includes an `html_safe` string for each queried service.
